### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by ronakshah2895**
Completed News Feed

### CODE REVIEW

**Comment by jainamdi** [comment]
Images in the news feed are not loading. Also, there was nothing mentioned in PR to install the new library 'react-scroll'. Please check. Also, add the version number of 'react-scroll' if required.

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -1,96 +1,114 @@
...
     const user = {
       email: this.state.email,
-      password: this.state.password
-    }
+      password: this.state.password,
+    };
 
-    login(user).then(res => {
+    login(user).then((res) => {
       if (res) {
```
Comment (shah-tejas) [inline src/components/Login/Login.js:30]: We need a check here to see if user is confirmed/active. If not, user should not be able to see his profile and a message "Please wait to get confirmed" should be shown here.

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -1,96 +1,114 @@
...
-      name="password"
-      value={this.state.password}
-      onChange={this.onChange}
-      />
-      </div>
-      <div className="form-check ml-5">
-      <input type="checkbox" class="form-check-input" name="Rememberme" />
-      <label class="label mb-1" forHTML="Rememberme">Remember Me</label>
-      </div>
+        <style>
```
Comment (shah-tejas) [inline src/components/Login/Login.js:39]: Avoid inline styles. Add all styles in respective scss files. Will help keep all styles maintainable. Also should avoid styling entire body element.
Wrap element to style in a div and apply style only in that div.

File: src/pages/landingpage/components/newsFeed/NewsFeed.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      height: 100%
+    }
+    .card-title {
+      font-size: 18px;
+      font-weight: bold;
+    }
+    .card-text {
+      font-size: 14px;
+
+      &.read-more {
```
Comment (shah-tejas) [inline src/pages/landingpage/components/newsFeed/NewsFeed.scss:23]: nice use of nesting :)

**Review by shah-tejas — APPROVED** [review]
Lgtm, some minor changes.

**Pull request merged**